crates/modelardb_server/src/metadata.rs

```diff
@@ -293,7 +228,7 @@ impl MetadataManager {
                     "INSERT OR IGNORE INTO model_table_hash_table_name (hash, table_name) VALUES ({}, '{}')",
                     signed_tag_hash, model_table_metadata.name
                 )
-                .as_str(),
+                    .as_str(),
```

Comment:
Unclear why additional whitespace has been added.

Reply:
It might have been introduced by some weird PyCharm formatting. It seems that `rustfmt` does not change anything no matter if there is whitespace or not so I have removed the added whitespace.